README.md

```diff
@@ -1,31 +1,61 @@
-# NF1_SchwannCell_data_analysis
-Here we analyze [NF1](https://www.sciencedirect.com/science/article/pii/B9780444529022000539) morphology data from the [nf1_cellpainting_data](https://github.com/WayScience/nf1_cellpainting_data) repository.
-Our analysis consists of finding interesting patterns in the NF1 cell morphology data from the [nf1_cellpainting_data](https://github.com/WayScience/NF1_SchwannCell_data_analysis) repository, and then developing, evaluating and interpretting our NF1 model.
-The NF1 model is a binary classifcation model optimized from NF1 cell morphology data to classify cell genotypes {Null, WT} for the NF1 gene.
+# NF1 Schwann cell data analysis
+
+We aim to characterize cell morphology signatures of neurofibromin in Schwann cells. 
+We applied a modified Cell Painting assay on two isogenic Schwann cell lines, one of the wildtype genotype (*NF1+/+*) and one of the null genotype (*NF1-/-*), both from the same patient.
+The modified assay stains for four organelles: nuclei, endoplasmic reticulum, mitochondria, and actin. 
+We applied CellProfiler pipelines to perform quality control, illumination correction, segmentation, and feature extraction. 
+
+![Figure 1](./3.figures/figures/main_figure_1_workflow.png)
```

Comment:
I think this figure does belong in the data repo since this will be the main repo that people will go to from the manuscript. The data repo will end up be being more of a supplemental or complimentary to the main analysis repo.

> Does it make sense to group segmentation and feature extraction in B of this figure?

Yes, it does because in CellProfiler, we perform both of these steps in the same pipeline. So it makes sense in the workflow to pair them instead of separating them which might introduce more white space.